src/CMakeLists.txt

```diff
@@ -98,9 +98,10 @@ set_target_properties(
     CLEAN_DIRECT_OUTPUT 1
     FOLDER "Core")
 
+# NOMINMAX is needed for Windows to circumvent issues with std::numeric_limits<T>
 set_property(TARGET phylanx_component APPEND
   PROPERTY COMPILE_DEFINITIONS
-  "PHYLANX_EXPORTS")
+  "PHYLANX_EXPORTS;NOMINMAX")
```

Comment:
Please set this on Windows platforms only. Also, the comment is misleading.

Reply:
How is the comment misleading? The code won't compile without it.

Reply:
I didn't say it would compile without NOMINMAX. I meant that the issue is not related to `std::numeric_limits<T>`. Th eproblem will occur whenever `min()` or `max()` are used in an unguarded way.